migrations/2022-07-07-182650_comment_ltrees/up.sql

```diff
@@ -60,7 +60,7 @@ ORDER BY
 	breadcrumb;
 
 -- Remove indexes and foreign key constraints, and disable triggers for faster updates
-alter table comment disable trigger all;
+alter table comment disable trigger user;
 
```

Comment:
Changing old migrations does scare me, but most people have gotten past this one, so its fine.

Reply:
I don't like it either but AFAICT the only way to fix the literal SQL of old migrations is to edit it directly. Also since the resulting database with and without this change should be identical, it should be fine.

Reply:
I agree that this is fine.